app/Livewire/Project/AddEmpty.php

```
@@ -37,7 +37,12 @@ public function submit()
                 'uuid' => (string) new Cuid2,
             ]);
 
-            return redirect()->route('project.show', $project->uuid);
+            $productionEnvironment = $project->environments()->where('name', 'production')->first();
```

PHPMD (2.15.0)

Avoid excessively long variable names like $productionEnvironment. Keep variable name length under 20. (undefined)

(LongVariable)